### ISSUE TITLE
italic text doesn't seem to work on mac

### ISSUE THREAD

**Issue by jrmuizel** [mystor/phlay#18]
When getting information about 'sitm'

```
Traceback (most recent call last):
  File "/Users/jrmuizel/bin/phlay", line 502, in <module>
    main()
  File "/Users/jrmuizel/bin/phlay", line 493, in main
    process_args(args)
  File "/Users/jrmuizel/bin/phlay", line 397, in process_args
    f"{style.italic}{buginfo['summary']}{style.reset}")
  File "/Users/jrmuizel/bin/phlay", line 34, in __getattr__
    return curses.tparm(curses.tigetstr(name)).decode('latin1')
TypeError: a bytes-like object is required, not 'NoneType'
```

**Comment by mystor** [comment]
Hopefully this is fixed in the new version from #16. I'd love feedback on it.

**Comment by mystor** [comment]
Fixed by #13
